typescript-advanced/tasks/005-x-is-of-type-unknown.problem.ts

```diff
@@ -6,6 +7,19 @@ const somethingDangerous = () => {
 
 try {
   somethingDangerous();
-} catch (error) {
+} catch (error: any) {
   console.log(error.message);
-}
\ No newline at end of file
+}
+
+// 2. Solution with type casting
+const anotherSomethingDangerous = () => {
+  if (Math.random() > 0.5) {
+    throw new Error("Oh dear!");
+  }
+};
+
+try {
+  somethingDangerous();
+} catch (error: unknown) {
```

Comment:
Suggestion of another solution:
try {
  somethingDangerous();
} catch (error) {
  if (error instanceof Error) {
    console.log(error.message);
  } else {
    throw error;
  }
}